packages/v4/src/content/design-guidelines/components/alert/alert.md

```diff
@@ -4,123 +4,134 @@ section: components
 related: ['Alert', 'Alert group']
 ---
 
-An **alert** is used to notify the user about a change in status or other event without blocking other actions in an interface. These guidelines talk about how to properly use different types of alerts in your designs.
+An **alert** is a notification that provides brief information to the user without blocking their workflow.
+
+There are two main types of alerts:  
+
+- **Inline alerts**: Inline alerts communicate information about a specific user action on a page. An inline alert appears within the content area and disappears when the user closes it or navigates away from the page.
+
+![inline notification placement](./img/inline-notification.png)
+
+- **Toast alerts**: Toast alerts communicate information about an update, confirmation, or other activity. A toast alert overlys content in the upper-right corner of the page and disappears when it times out or when the user dismisses it.
+
+![toast notification placement](./img/toast-notification.png)
 
 ## Elements
-### Alert
-Two styles can be applied to an alert depending on whether it is intended as an [inline](#inline-alerts) or [toast](#toast-alerts) alert. Toast alerts should always be dismissable, whereas inline alerts can be both dismissable and non-dismissable. Other elements are consistent between alert types.
+Toast alerts are always dismissible, and inline alerts can be both dismissable and non-dismissible. Other elements are consistent between both alert types.
 
 <img src="./img/alert-types.png" alt="Alert types" />
 
-1. **Status area:** Icon and color indicate the status or severity of the alert.
-2. **Alert title:** Include a short descriptive message. Long message text will wrap. Text color is coded to match alert type.
-3. **Close (optional):** Dismiss the alert. Required for toast alerts and optional for inline alerts.
-4. **Actions (optional):** One or more actions can be placed inside of an alert.
-5. **Description (optional):** If a longer message is required, it can be included within the Description field. This can also include a navigation link as an alternative to an action.
+1. **Status area**: The status area indicates the severity of the alert with a specific icon and color.  
```

Comment:
Is there a reason you spaced these out the way you did (ie: blank line between each number)? They're not indenting as a result - so it's not formatting like a typical numbered list!

Reply:
When I didn't have the spacing, they all meshed together in a paragraph. I can retry to see if I can fix that.